packages/design-system/src/components/toast.tsx

```diff
@@ -130,3 +131,22 @@ export const Toaster = () => {
 };
 
 export { toast };
+
+export const toastNonFieldErrors = (
```

Comment:
1. not sure if this should be here on the level of design system
2. design system doesn't know zod yet and I am not sure it should
3. I don't understand the semantics of "toastNonFieldErrors"

Reply:
> not sure if this should be here on the level of design system

Yeah, probably better to put it in `shared` in the app.

> I don't understand the semantics of "toastNonFieldErrors"

"Given form errors from Zod, show a toast for each error that does not correspond to a known field"

Reply:
ah so the semantics is this: toastUnknownFieldErrors

Reply:
Looks like we could use shared/form-utils or something

Reply:
`toastUnknownFieldErrors` wouldn't be right because it shows 2 kinds of errors:

1. Errors of entire form
2. Errors of fields that we didn't expect to receive (don't have fields for in UI)

So the name would fit for `#2` but not `#1`